### PR TITLE
Update pipeline-operator documentation

### PR DESCRIPTION
**Pull request by mAAdhaTTah**
Related to https://github.com/babel/babel/pull/8196

### CODE REVIEW

**Comment by hzoo** [comment]
Deploy preview for *babel-new* ready!

Built with commit f87d2e4c8f642b91e264783aceb0e2e726cf8e00

https://deploy-preview-1700--babel-new.netlify.com

File: docs/plugin-proposal-pipeline-operator.md [hunk 1]
```diff
@@ -18,10 +18,18 @@ $ npm install @babel/plugin-proposal-pipeline-operator
 
 ```json
 {
-  "plugins": ["@babel/plugin-proposal-pipeline-operator"]
+  "plugins": [["@babel/plugin-proposal-pipeline-operator", { "proposal": "minimal" }]]
 }
 ```
 
+The Pipeline Proposal is one of three competing implementations. Which implementation the plugin should use is configured with the `"proposal"` option. This option is required and should be one of:
```
Comment (hzoo) [inline docs/plugin-proposal-pipeline-operator.md]: general question: should we start adding versions in docs? Like mention this is only the case for >= beta.51, and only merge docs PRs after the release?

Comment (mAAdhaTTah) [reply docs/plugin-proposal-pipeline-operator.md]: Do we need to specify by beta version for something like this?

**Comment by nicolo-ribaudo** [comment]
You should mention that currently only minimal is supported

**Comment by mAAdhaTTah** [comment]
@nicolo-ribaudo Fixed!

**Pull request merged**